### DISCUSSION TITLE
Compare v2.0 - discussion invitation

### DISCUSSION THREAD

**Discussion by AnaBelgun** [TerriaJS/terriajs#6155] [Ideas]
We are preparing to release a new version of the Compare functionality (formerly know as the 'map splitter'). Here is a video with the work in progress, We're pretty close to releasing, but we'd like to hear if there are any major concerns to consider. 
https://youtu.be/uwvvySJA2y4

**Comment by zoran995** [comment]
Happy to see RFC, and pretty sure that you knew that I will be the first to comment 😆. I spent some time playing around with new compare workflow and also thought a lot about it since last time commenting on pull request. I see how this will be easier for new users to start working with, but personally I find it lacking some stuff. I also realize how old compare can be confusing at the start but when you get used to it, it becomes a killer feature that can't be replaced easily.

IMHO, the most important feature in compare tool is having the possibility to add multiple datasets per side with a possibility to easily change the order of them (in the specified side). This was really easy with the old compare tool, but the new workflow, at least for now, forces only one dataset per side. 

Some other stuff that I noticed
- No ability to hide dataset chosen for one side and collapse the dataset (useful with long legends). i.e. need to check something on basemap or dataset shown over the entire map.
- What is the order of the dataset when we start compare, datasets added to the side are moved to the top in the workbench?
- When the compare is stared dataset shown over the entire map will preserve their show/hide or hidden by default?
- if new compare is chosen as default how user can reorder datasets without losing chosen datasets in the compare tool. i.e. I have 5 datasets on the map and enter the workflow, choose the desired dataset for each side date, and then realize I need to change the order of datasets shown over the entire map. Currently, the user has to close compare change order of datasets and run compare tool from scratch.

Given the above, I would propose calling this diff tool and leaving compare with existing support, but that name is already taken.

**Comment by AnaBelgun** [comment]
Hi @zoran995 
Thank you very much. As usual, your feedback is sensible and much appreciated by our team. 
We are considering all the issues/edge cases you raised and have another go at designing those steps. 
We'll get in touch when we have some options to address your points.
